get_mock_server.sh

```diff
@@ -1,7 +1,9 @@
 #!/bin/bash
-
-if ! [ -e $1/mock_server-x64-linux ]; then
-  curl -L -o $1/mock_server-x64-linux https://github.com/googleinterns/cloud-operations-api-mock/raw/master/cmd/mock_server-x64-linux
-  chmod +x $1/mock_server-x64-linux
+VERSION=v1-alpha
+if ! [ -e $1/mock_server-x64-linux-$VERSION ]; then
+  curl -L -o $1/mock_server-x64-linux-$VERSION https://github.com/googleinterns/cloud-operations-api-mock/releases/download/v1-alpha/mock_server-x64-linux-$VERSION
+  chmod +x $1/mock_server-x64-linux-$VERSION
 fi
 
+ln -sf $1/mock_server-x64-linux-$VERSION $1/mock_server
```

Comment:
Looks like another variable `BINARY=mock_server-x64-linux-$VERSION` would clean this up

Reply:
@mabdi3

Reply:
ah I think this comment got lost. I think it would clean it up as well. Done.